support-frontend/app/controllers/Application.scala

```diff
@@ -234,6 +239,45 @@ class Application(
     )()).withSettingsSurrogateKey
   }
 
+  def contributionsCheckoutInEpic(countryCode: String): Action[AnyContent] = PrivateAction { implicit request =>
+    implicit val settings: AllSettings = settingsProvider.getAllSettings()
+
+    val geoData = request.geoData
+    val serversideTests = generateParticipations(Nil)
+    val uatMode = testUsers.isTestUser(request)
+    val guestAccountCreationToken = request.flash.get("guestAccountCreationToken")
+
+    Ok(
+    views.html.contributions(
+      title = "Contributions checkout in Epic",
+      id = s"contributions-landing-page-in-epic-$countryCode",
+      mainElement = assets.getSsrCacheContentsAsHtml("contributions-checkout-in-epic", "contributions-checkout-in-epic.html"),
+      js = Left(RefPath("contributionsCheckoutInEpic.js")),
+      css = Right(StyleContent(Html(""))),
```

Comment:
All the styles are done via `emotion` so I don't need any style sheets. This does result in an empty `style` tag being added to the page

Reply:
I don't love that it does that, but this felt like the quickest/easiest solution

Reply:
This is no longer empyt as it was a convenient place to remove the default margins added to the body element!